locations/commands/sd.py

```diff
@@ -94,5 +95,4 @@ def run(self, args, opts):
         if stats_dict.get("item_scraped_count", 0) == 0:
             print("failed to decode structured data")
         if opts.stats:
-            for k, v in stats_dict.items():
-                print(k + ": " + str(v))
+            print(pprint.pformat(stats_dict))
```

Comment:
```suggestion
            pprint.pprint(stats_dict)
```

May be more readable?